tests/integration/questionnaire/test_questionnaire_final_confirmation.py

```diff
@@ -0,0 +1,68 @@
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestQuestionnaireFinalConfirmation(IntegrationTestCase):
+
+    def test_final_confirmation_asked_at_end_of_questionnaire(self):
+        base_url = '/questionnaire/test/final_confirmation/789/'
+
+        # Given
+        token = create_token('final_confirmation', 'test')
+        self.client.get('/session?token=' + token.decode(), follow_redirects=False)
+
+        # When
+        # We proceed through the questionnaire
+        post_data = {
+            'action[start_questionnaire]': 'Start Questionnaire'
+        }
```

Comment:
To back my point up above this test would be smaller and simpler if the schema only had one question as the questions are not the focus of the test